packages/core/contracts/zodiac/OptimisticGovernor.sol

```diff
@@ -318,4 +310,12 @@ contract OptimisticGovernor is Module, Lockable {
     function _sync() internal {
         optimisticOracle = _getOptimisticOracle();
     }
+
+    function _getOptimisticOracle() private view returns (OptimisticOracleInterface) {
+        return OptimisticOracleInterface(finder.getImplementationAddress(OracleInterfaces.OptimisticOracle));
+    }
+
+    function _isContract(address addr) private view returns (bool isContract) {
```

Comment:
I don't think you need to name the `bool` return value here. But if you want to keep it named you could just change the body to be `isContract = addr.code.length > 0`. But I lean toward deleting `isContract`